frictionless/schema/field.py

```diff
@@ -263,6 +280,12 @@ def metadata_validate(cls, descriptor: IDescriptor):  # type: ignore
             field = Class(
```

Comment:
I think we can just try `field = Class.from_descriptor(descriptor)` to make it work in general

Reply:
It does not work with `from_descriptor`, it leads to a `RecursionError`.